lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
```

Comment:
Maybe I'm missing the point, but it's not redefined at those lines, it just modified with the possible `spacingTop` parameter.